examples/separate-config/every-feature-multiple-dest-single-ms.html

```diff
@@ -0,0 +1,96 @@
+                    <option value="6">Lisp</option>
+                    <option value="7">Lua</option>
+                    <option value="8">MATLAB</option>
+                    <option value="9">NewLISP</option>
+                    <option value="10">PHP</option>
+                    <option value="11">Perl</option>
+                    <option value="12">SQL</option>
+                    <option value="13">Unix shell</option>
+                </select>
+            </div>
+            
+            <div class="col-sm-2">
+                <button type="button" id="multi_d_undo" class="btn btn-primary btn-block">Undo</button>
+                <button type="button" id="multi_d_redo" class="btn btn-warning btn-block">Redo</button>
+                <hr style="margin: 40px 0 60px;" />
+                <button type="button" id="multi_d_rightAll_1" data-context="#multi_d_to_1" class="btn btn-default btn-block"><i class="glyphicon glyphicon-forward"></i></button>
+                <button type="button" id="multi_d_rightSelected_1" data-context="#multi_d_to_1" class="btn btn-default btn-block"><i class="glyphicon glyphicon-chevron-right"></i></button>
+                <button type="button" id="multi_d_leftSelected_1" data-context="#multi_d_to_1" class="btn btn-default btn-block"><i class="glyphicon glyphicon-chevron-left"></i></button>
+                <button type="button" id="multi_d_leftAll_1" data-context="#multi_d_to_1" class="btn btn-default btn-block"><i class="glyphicon glyphicon-backward"></i></button>
+                <hr style="margin: 40px 0 60px;" />
+                <button type="button" id="multi_d_rightAll_2" data-context="#multi_d_to_2" class="btn btn-default btn-block"><i class="glyphicon glyphicon-forward"></i></button>
+                <button type="button" id="multi_d_rightSelected_2" data-context="#multi_d_to_2" class="btn btn-default btn-block"><i class="glyphicon glyphicon-chevron-right"></i></button>
+                <button type="button" id="multi_d_leftSelected_2" data-context="#multi_d_to_2" class="btn btn-default btn-block"><i class="glyphicon glyphicon-chevron-left"></i></button>
+                <button type="button" id="multi_d_leftAll_2" data-context="#multi_d_to_2" class="btn btn-default btn-block"><i class="glyphicon glyphicon-backward"></i></button>
+            </div>
+
+
+
+
+            <div class="col-xs-5">
+                <b>Known languages</b>
+                <select name="to[]" id="multi_d_to_1" class="form-control palette" size="8" multiple="multiple">
+                    <option value="1">C++</option>
+                    <option value="2">C#</option>
+                    <option value="14">ABCDE</option>
+                </select>
+                <div class="row">
+                    <div class="col-sm-6">
+                        <button type="button" id="multi_d_moveUp_1" data-context="#multi_d_to_1" class="btn btn-default btn-block"><i class="glyphicon glyphicon-arrow-up"></i></button>
```

Comment:
The moveUp/Down buttons need context with multiple destinations, so that items are not moved between the destinations.